applications/ULFapplication/ULF_application.cpp

```diff
@@ -19,31 +19,78 @@
 #include "includes/define.h"
 
 #include "ULF_application.h"
-//#include "includes/variables.h"
+#include "includes/variables.h"
+#include "includes/deprecated_variables.h"
+
 #include "geometries/triangle_2d_3.h"
 #include "geometries/tetrahedra_3d_4.h"
 #include "geometries/point_3d.h"
 #include "geometries/point_2d.h"
 
+#include "geometries/triangle_3d_3.h"
+#include "geometries/quadrilateral_2d_4.h"
+#include "geometries/hexahedra_3d_8.h"
+#include "geometries/line_2d.h"
+#include "geometries/line_2d_3.h"
+
 namespace Kratos
 {
 
-/*	KRATOS_CREATE_VARIABLE(double, NODAL_AREA)
-	KRATOS_CREATE_VARIABLE(double, NODAL_H)
-	KRATOS_CREATE_VARIABLE(double, IS_STRUCTURE)
-	KRATOS_CREATE_VARIABLE(double, IS_FLUID)
-	KRATOS_CREATE_VARIABLE(double, IS_BOUNDARY)
-	KRATOS_CREATE_VARIABLE(double, IS_FREE_SURFACE)
-	KRATOS_CREATE_VARIABLE(double, IS_FREE_SURFACE)
-	KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(NORMAL_TO_WALL)
-*/
-//KRATOS_CREATE_VARIABLE(double, IS_LAGRANGIAN_INLET)
-
+//     KRATOS_CREATE_VARIABLE(double, IS_LAGRANGIAN_INLET)
+    //KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(PRESSURE_FORCE)
+    //KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(DISP_FRAC)
+    KRATOS_CREATE_VARIABLE(double, PRESSURE_OLD_IT)
```

Comment:
This variable already exists. Indeed, multiple variables in here already exist. As far as I can understand this might be a potential problem...

Reply:
Greetings dear @rubenzorrilla,

I just check about the variables added in the ULF applications, and nothing is duplicated with the "variables.h" or "variables.cpp" inside "kratos/kratos/includes" or "kratos/kratos/sources". However, this "PRESSURE_OLD_IT" for example is defined inside the cfd_variables... and "PRESSURE_FORCE" is defined inside the meshing_applications,,,, so what do you recommend in this case please

Reply:
You have two options in here, 
- Include the files that declare these variables.
- Or defining a new local variable in your application.
For the PRESSURE_OLD_IT, I would include the cfd_variables file since it belongs to the core. For the PRESSURE_FORCE, do whatever you want, but in any case, do not declare it again.

Reply:
thanks dear, 

sure will do,

so I will go with the first option always,

Reply:
Greetings dear @rubenzorrilla ,

I hope I have done the required as required, kindly note that I have I removed the variables that are already existing, and also I defined the needed variables fir this code locally inside ULF applications. Also, I tried to follow the naming conventions for the added files, with the recommended header format,

Thanks and best regards,